### ISSUE TITLE
Setting Labels using GraphML

### ISSUE THREAD

**Issue by ftrotter** [gephi/gephi#1502]
Hello. 

When importing GraphML files, I am having trouble setting the label of nodes. So far I have tried three things, based on hints in the various documentation(s). the two unorthodox things I have tried is to add a "label" attribute to the <node> element. I have also tried adding a <data> element with a key of "label". Neither of those seems to work correctly. 

[As per the graphml standard](http://graphml.graphdrawing.org/specification/element_desc.xsd.htm), the way this should work is using a single <desc> element as the first sub-element of the <node> element. That does not seem to work either. The desc element is not very well outlined in the [GraphML primer](http://graphml.graphdrawing.org/primer/graphml-primer.html) but it is there, at the very end. 

Personally I think that the labels should be set a per the GraphML standard, but I would be happy to conform to what gephi is expecting, but there is no mention of setting the labels properly in [this gephi documentation](https://gephi.org/users/supported-graph-formats/graphml-format/). 

Thank you all for working on such a great project. 

-Fred Trotter

**Comment by eduramiba** [comment]
Should be fixed with #1719 Please try with latest 0.9.2 snaphost https://github.com/gephi/gephi#nightly-builds

**Comment by krishna116** [comment]
I meet the same question about how to set the label, and I find a method: define custom-label-property in the graphml scope and use the custom-label-property in the node scope.  
For example:
```
<graphml>
  <key id="label" for="node" attr.name="label" attr.type="string"/> <!--Define custom label property-->
  
  <graph id="G" edgedefault="directed">
    <node id="1">
        <data key="label">a</data>      <!--Using custom label property-->
    </node>
    <node id="2">
        <data key="label">b</data>      <!--Using custom label property-->
    </node>
    <node id="3">
        <data key="label">c</data>      <!--Using custom label property-->
    </node>
    <edge source="1" target="2"/>
    <edge source="1" target="3"/>
  </graph>
</graphml>
```
I don't know about gephi but I know yEd has given an option to mapping any custom property to label so it can be rendered.
![image](https://user-images.githubusercontent.com/41564221/174413584-5fcfb238-4521-40a0-b539-492462e3b8e3.png)
That's all, thanks.
